### DISCUSSION TITLE
계정관련하여 질문할게 있습니다.!

### DISCUSSION THREAD

**Discussion by phphll** [fusiondevkr/pl900#42] [1 Q&A]
다름이 아니라 제가 사용하던 계정이 곧 계약이 만료되어 계정이 사라질거 같은데

혹시 제가 가진 개발자 계정으로 등록 가능할까요 ?

아니면 상관 없으려나요 ....

개발자 계정은 phphll@phphll.onmicrosoft.com 입니다.

**Comment by justinyoo** [comment]
M365 개발자 계정은 3개월간 사용이 가능합니다. 그동안 파워 앱, 파워 오토메이트 등 하나라도 퍼블리시를 하셨다면 자동으로 기한 연장이 가능해요. 저희가 드린 숙제만 꾸준히 해 보셨다면 큰 문제 없이 계속 사용이 가능합니다!
